json-schema-provider/src/main/java/io/confluent/kafka/schemaregistry/json/schema/SchemaTranslator.java

```diff
@@ -486,6 +482,12 @@ public SchemaContext visitPropertySchema(String property,
         .addPropertySchema(property, ctx.schema()));
   }
 
+  @Override
+  public SchemaContext visitReadOnlySchema(ReadOnlySchema schema) {
```

Comment:
this will be inherited from the parent, right?

Reply:
This overridden method just calls super at the moment